app/views/file_uploads/product_survey.html.haml

```diff
@@ -72,13 +72,11 @@
     = f.error_messages
     = f.hidden_field :file_type
 
-    %ul.form
-      %li
-        = f.label :file, "File", class: "required"
-        = f.file_field :file
+    = f.label :file, "File", class: "required"
+    = f.file_field :file
 
     %ul.inline
-      %li= f.submit "Upload", data: { disable_with: "Uploading..." }, class: "btn"
+      %li= f.submit text("shared.upload"), data: { disable_with: "Uploading..." }, class: "btn"
```

Comment:
There's a translation for "Uploading..." at `file_uploads.index.submitted` (which is maybe not the ideal spot for it)